NetCoreMentoring/src/NetCoreMentoring.App/Controllers/CategoryController.cs

```diff
@@ -37,5 +38,24 @@ public IActionResult Index()
             }
         }
 
+        [ServiceFilter(typeof(ImageCacheFilter))]
```

Comment:
the idea of the test is to use middleware, so any other solution is not applicable in this particular task